### ISSUE TITLE
Mismatch between LICENSE file and the source files' copyright headers

### ISSUE THREAD

**Issue by paulapatience** [bbopt/nomad#52]
The LICENSE file states that NOMAD is licensed under [LGPL-3.0-only](https://spdx.org/licenses/LGPL-3.0-only.html), but the file headers refer to [LGPL-3.0-or-later](https://spdx.org/licenses/LGPL-3.0-or-later.html). This was discovered by Ludovic Courtès and brought to my attention [here](https://mail.gnu.org/archive/html/guix-patches/2021-07/msg01151.html).

**Comment by nckx** [comment]
*Edit: Ludovic Courtès [already said everything I did below](http://issues.guix.gnu.org/49476#3).  I'm confident that this bug can be closed.*

Paul,

Thanks for double-checking licencing!  However, there does not appear to be a problem here.  `LICENSE` is simply the text of the LGPL version 3. [It should not be changed.](https://www.gnu.org/licenses/gpl-faq.en.html#ModifyGPL).

(Are you using some tool that sniffs licences?  The string "-only" does not appear in the current `LICENSE` file at all.)

> The LICENSE file states that NOMAD is licensed under LGPL-3.0-only

That's [not true](https://github.com/bbopt/nomad/blob/master/LICENSE#L156):
```
                                     [...] If the Library as you
received it does not specify a version number of the GNU Lesser
General Public License, you may choose any version of the GNU Lesser
General Public License ever published by the Free Software Foundation.
```

Not even only later ones!  Here, upstream does specify a version in both the top-level `README` and all the `.hpp` files I happened to check (I'm reduced to using the Web interface), which explicitly limit you to "version 3 or later".  This is excellent practice.
